pylint/lint/pylinter.py

```diff
@@ -745,7 +754,10 @@ def _lint_files(
         check_astroid_module: Callable[[nodes.Module], bool | None],
     ) -> None:
         """Lint all AST modules from a mapping.."""
+        progress_reporter = get_progress_reporter(self.verbose, len(ast_mapping))
+        progress_reporter.print_message(f"Linting {len(ast_mapping)} modules.")
```

Comment:
I think this should probably be called `start_linting` so that the reporter itself can determine what kind of message it wants to print here.

Reply:
I kept it general for no real reason, but sure. :-)

Reply:
Actually, print_message might still be better as this same message is used to "get ASTs" ... unless you wanted to add that method get_asts() as well.  Basically all that would do is move code from one area to another, which might be what you're getting after.

Reply:
Ah whoops, re my "Actually, print_message might still be better as this same message is used to "get ASTs" " -- but then you said later that it should have its own method. 👍

Reply:
Pushed, now is `progress_reporter.start_linting()`